### DISCUSSION TITLE
Thumbnail Storage

### DISCUSSION THREAD

**Discussion by Arindrew** [immich-app/immich#5638] [Q&A] [answered]
Where are thumbnails stored? Locally in the docker container? I am worried than when I import my photos, the thumbnail storage is going to exceed the size of my container (restricted in unraid to 20G - along with all the other dockers I have). 

Is it possible to configure an external volume that thumbnails can be stored?

**Comment by alextran1502** [comment, marked as answer]
it is stored in `UPLOAD_LOCATION` under the `thumb` directory, so not in your Unraid container

**Reply by Arindrew** [reply]
Does that also apply to thumbnails generated from images in external storage?

**Reply by bo0tzz** [reply]
Yes
